### ISSUE TITLE
Firehorse Integration

### ISSUE THREAD

**Issue by leosol** [bkerler/edl#80]
Hei guys, do you think that this project could be used with AlepthSecurity Firehorse tool? Or would it be out of the scope of the project?
Link to Alepth Security's Firehorse
https://github.com/alephsecurity/firehorse

**Comment by bkerler** [comment]
Yes, of course it can be used with Firehorse. You just need to replace the qfil references with this tool and with option "rawxml". Or even better, use the tcpip functionality. Just run using ./edl.py server. See examples for a tcp ip example client.

**Comment by leosol** [comment]
Sounds easy! I will try. I'm with a Redmi Note 5A which I would like to try firehorse stuff. Could replicate Aleph's memory based attack on it (https://github.com/leosol/storage-based-edl/tree/main/redmi-note-5a), but could not advance in all what they have found.
Do you know any body who got any success with firehorse? I mean that memory based attack / runtime debugger?
